pages/common/tcsh.md

```diff
@@ -23,6 +23,10 @@
 
 `tcsh -n {{path/to/script.tcsh}}`
 
+- Execute specific commands from stdin:
```

Comment:
This would be more accurate.
```suggestion
- Execute a specific command from `stdin`:
```

Reply:
I am not sure whether the first part of suggestion makes page better. Just one command is demonstrated in placeholder for simplicity but in general many commands can be run.

Reply:
But the goal is to describe what an example does, not what it could do. I think it's a little confusing, when the description doesn't match the example

Reply:
I guess I refresh all shell pages again soon and address this comment. Possible solution: introduce new generic placeholder and make it recognizable by our TlDr page parsers.

Stay tuned. 😄